### PR TITLE
Test migration phase 8

### PR DESCRIPTION
**Pull request by sergeybykov**

### CODE REVIEW

File: src/TesterInternal/ClientConfig_AzureStreamProviders.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <TraceLevelOverride LogPrefix="Runtime" TraceLevel="Info" />
+    <TraceLevelOverride LogPrefix="Application" TraceLevel="Info" />
+    <TraceLevelOverride LogPrefix="Tetris" TraceLevel="Info" />
+    <TraceLevelOverride LogPrefix="AssemblyLoader" TraceLevel="Warning" />
+  </Tracing>
+  <Statistics MetricsTableWriteInterval="300s" PerfCounterWriteInterval="30s" LogWriteInterval="300s" StatisticsCollectionLevel="Info"/>
+  <Messaging ResponseTimeout="30s" ClientSenderBuckets="8192" MaxResendCount="0" UseMessageBatching="false" MaxMessageBatchingSize="100"/>
+  <StreamProviders>
+    <Provider Type="Orleans.Providers.Streams.SimpleMessageStream.SimpleMessageStreamProvider" Name="SMSProvider" FireAndForgetDelivery="false"/>
+    <!--<Provider Type="Orleans.Providers.Streams.AzureQueue.AzureQueueStreamProvider" Name="AzureQueueProvider" DataConnectionString="DefaultEndpointsProtocol=https;AccountName=plays3orleans1;AccountKey=XbYEMjNA1e79ZTe/ORoFXR5Or4m97cckkcKMTTHauZ3FtL1u9GnJ2rLX7DJMZ7fmCCBdgH6LySkj0O2vACDugw=="/>-->
```
Comment (jdom) [inline src/TesterInternal/ClientConfig_AzureStreamProviders.xml]: account leaked, please roll the key

File: src/TesterInternal/MockStorageProvider.cs [hunk 2]
```diff
@@ -1,161 +0,0 @@
-﻿using System;
```
Comment (jdom) [inline src/TesterInternal/MockStorageProvider.cs]: dang it, was scratching my head thinking how would some tests compile without this class, until I noticed these were loose files not referenced by a project

File: src/TesterInternal/ClientConfig_AzureStreamProviders.xml (hunk 1 above)
Comment (sergeybykov) [inline src/TesterInternal/ClientConfig_AzureStreamProviders.xml]: This is an old key that was rolled a long time ago. But good catch regardless.

File: src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs [hunk 3]
```diff
@@ -37,12 +37,18 @@ public class PersistenceGrainTests_AzureStore : UnitTestSiloHost
         private const int MaxReadTime = 200;
         private const int MaxWriteTime = 2000;
 
+        private static readonly Guid initialServiceId = Guid.NewGuid();
+
         private static readonly TestingSiloOptions testSiloOptions = new TestingSiloOptions
         {
             SiloConfigFile = new FileInfo("Config_AzureTableStorage.xml"),
```
Comment (jdom) [inline src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs]: I see this Config_AzureTableStorage.xml file being used, but it was removed from the TesterInternal.csproj file at the bottom of the diff. Is it by design?

Comment (sergeybykov) [inline src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs]: You are right, this got messed up somehow. I'm adding the file back to the csproj.

Comment (sergeybykov) [inline src/TesterInternal/StorageTests/PersistenceGrainTests_AzureStore.cs]: Done. Pushed.

**Pull request merged**